app/models/draw_suites_update.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+#
+# Form / service object to handle the updating of draw suites
+class DrawSuitesUpdate
+  include ActiveModel::Model
+
+  attr_reader :size, :suite_ids, :drawn_suite_ids, :undrawn_suite_ids
+
+  # permit :update to be called on the parent class
```

Comment:
It's not an instance, that would be `def update`, but it's also not the parent class. I'll update it to `class object`.